portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects - Remus Neatu</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  
+  <body class="background">
+    <div class="menu">  
+        <a href='index.html'><b>Home</b></a>
+        <a><b>About</b></a>
+        <a href='projects.html'><b>Projects</b></a>
+        <a href='contact.html'><b>Contact</b></a>
+    </div>
+
+    <section id="dashboard">
+        <h1 class ="project">Projects</h1>
+        <a href="#firstProject">Go to first project</a>
+        <a href="#secondProject">Go to second project</a>
+        <a href="#thirdProject">Go to third project</a>
+    </section>
+   
+    <section id="firstProject">
+        <h1 class ="project">WorldWide Knowledge</h1>
```

Comment:
Worldwide

Reply:
Actually this is how I named my application. I did this only to obtain the WWK abreviation.